grr/server/grr_response_server/flows/general/osquery_test.py

```diff
@@ -155,17 +161,18 @@ def setUp(self):
     super(FakeOsqueryFlowTest, self).setUp()
     self.client_id = self.SetupClient(0)
 
-  def _InitializeFlow(self, query):
+  def _InitializeFlow(self, query: str, file_collect_columns: List[str]):
```

Comment:
If majority of calls doesn't care about this parameter, I would just make it optional.

Reply:
Please see my comment above: https://github.com/google/grr/pull/901#discussion_r545128095

Reply:
Ah, terrible GitHub review flow strikes again.

No strong opinion here, can stay as it is.

Reply:
Ah, terrible GitHub review flow strikes again.

No strong opinion here, can stay as it is.